types/indexer/indexer.go

```diff
@@ -234,7 +234,7 @@ func byteArrayToData[LD tinypack.LengthDescriptor](in []byte) (*primitives.Data[
 			return nil, err
 		}
 	}
-	p := primitives.DataFromHex[LD](sb.String())
+	p := primitives.MustDataFromHex[LD](sb.String())
```

Comment:
This function (`byteArrayToData`) does very stupid thing - it converts byte-array to hex, and then converts hex to data (which is byte-array inside).
So let's just remove this function and use `Data*FromHex` from all invocation places instead